### PR TITLE
Update the manual Ansible stages

### PR DESCRIPTION
**Pull request by tomassedovic**
This aligns the stages directory contents to the new stages we created
on master. That is: prep, pre-introspection, pre-deployment and
post-deployment.

These stages show up in the legacy API and the UI folks requested us to
do this.

### CODE REVIEW

**Comment by tomassedovic** [comment]
@mandre @jtomasek  I have not tested running the new stages. My setup is busted, but I created the PR so you can review (and possibly verify) it while I rebuild my tripleo environment.

File: ansible-tests/stages/pre-deployment.yaml [hunk 1]
```diff
@@ -1,18 +1,13 @@
...
+- hosts: all
   vars:
     metadata:
       name: Pre-deployment
-      description: Validations that run before any deployment
+      description: Validations that are run right before deploying the overcloud
       stage: pre-deployment
+- include: ../validations/512e.yaml
 - include: ../validations/discovery_diff.yaml
 - include: ../validations/undercloud-disk-space.yaml
```
Comment (mandre) [inline ansible-tests/stages/pre-deployment.yaml]: This one is not in the correct group.

**Comment by mandre** [comment]
LGTM, apart from the undercloud-disk-space.yaml validation placed in the wrong group, which is not really a big issue.

**Comment by tomassedovic** [comment]
@mandre good catch, thanks!

**Comment by mandre** [comment]
:+1: 
Feel free to merge after you've tested in your environment.

**Comment by tomassedovic** [comment]
Thanks! Will probably do that tomorrow.

**Comment by jtomasek** [comment]
+1, tested, works as expected

**Comment by tomassedovic** [comment]
Thanks @jtomasek!

**Comment by tomassedovic** [comment]
Tested. Merging.

**Pull request merged**